src/test/librados_test_stub/TestClassHandler.cc

```diff
@@ -43,7 +45,9 @@ void TestClassHandler::open_class(const std::string& name,
 void TestClassHandler::open_all_classes() {
   assert(m_class_handles.empty());
 
-  DIR *dir = ::opendir(".libs");
+  const char* env = getenv("CEPH_LIB");
+  std::string CEPH_LIB(env ? env : "lib");
+  DIR *dir = ::opendir(CEPH_LIB.c_str());
   if (dir == NULL) {
```

Comment:
@trociny I've made the appropriate adjustments in https://github.com/ceph/ceph/pull/8770. I did end up changing "lib" to ".libs" since that is a good default that CEPH_LIB should revert back to for the time being